### ISSUE TITLE
ListenBrainz import exceptions

### ISSUE THREAD

**Issue by iamdjk** [krateng/maloja#184]
Hello, I am trying to import my ListenBrainz data into Maloja (on Ubuntu 22.04) but the import function is throwing exceptions.  Please see the following from the command line:

```
usr@device:~$ maloja import /home/usr/Downloads/usr_lb-2023-01-29.json
[spotify] Successfully authenticated with Spotify
Parsing /home/usr/Downloads/usr_lb-2023-01-29.json as ListenBrainz export
This could take a while...
Imported 1000 scrobbles...
Traceback (most recent call last):
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/dbcache.py", line 47, in outer_func
    return cache[key]
KeyError: ('[{"artists": ["Sevdaliza"], "title": "Angel", "length": 422}]', '{}', <function connection_provider.<locals>.wrapper at 0x7f003c204820>, None, None)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/dbcache.py", line 47, in outer_func
    return cache[key]
KeyError: ('["Sevdaliza"]', '{}', <function connection_provider.<locals>.wrapper at 0x7f003c2049d0>, None, None)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/usr/.local/bin/maloja", line 8, in <module>
    sys.exit(main())
  File "/home/usr/.local/lib/python3.10/site-packages/doreah/control.py", line 51, in wrapper
    result = func(*args,**kwargs)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/__main__.py", line 178, in main
    return action(*args,**kwargs)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/proccontrol/tasks/import_scrobbles.py", line 101, in import_scrobbles
    add_scrobbles(scrobblebuffer)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 120, in wrapper
    return func(*args,**kwargs)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 255, in add_scrobbles
    ops = [
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 257, in <listcomp>
    **scrobble_dict_to_db(s,dbconn=dbconn)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 221, in scrobble_dict_to_db
    "track_id":get_track_id(info.get('track'),dbconn=dbconn),
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/dbcache.py", line 49, in outer_func
    result = inner_func(*args,**kwargs,dbconn=conn)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 115, in wrapper
    return func(*args,**kwargs)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 294, in get_track_id
    artist_ids = [get_artist_id(a,dbconn=dbconn) for a in trackdict['artists']]
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 294, in <listcomp>
    artist_ids = [get_artist_id(a,dbconn=dbconn) for a in trackdict['artists']]
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/dbcache.py", line 49, in outer_func
    result = inner_func(*args,**kwargs,dbconn=conn)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 115, in wrapper
    return func(*args,**kwargs)
  File "/home/usr/.local/lib/python3.10/site-packages/maloja/database/sqldb.py", line 346, in get_artist_id
    op = DB['artists'].select(
TypeError: FromClause.select() takes 1 positional argument but 2 were given
[dbcache] Cache: Size: 0 | Hits: 0/2 | Estimated Memory: 72 B
[sqldb] Database Cleanup...
[dbcache] Entity Cache: Size: 0 | Hits: 0/0 | Estimated Memory: 72 B
[dbcache] System RAM Utilization: 33.0%
[sqldb] Database Cleanup complete!
```

The process then hangs without completing from this point.  Please let me know if you need anything further to troubleshoot this issue.  Thank you!

**Comment by iamdjk** [comment]
Upon attempting to import data from last.fm, I encountered the same issue.  Maybe my installation was not set up properly?

**Comment by krateng** [comment]
Should also be a compatibility issue with SQLAlchemy 2.0 (see #196), fixed by 3ae395f.
